go/cmd/orchestrator/main.go

```diff
@@ -126,41 +115,17 @@ func main() {
 	config.RuntimeCLIFlags.ConfiguredVersion = AppVersion
 	config.MarkConfigurationLoaded()
 
-	if len(flag.Args()) == 0 && *command == "" {
-		// No command, no argument: just prompt
-		fmt.Println(app.AppPrompt)
-		return
-	}
-
 	helpTopic := ""
 	if flag.Arg(0) == "help" {
 		if flag.Arg(1) != "" {
 			helpTopic = flag.Arg(1)
 		}
-		if helpTopic == "" {
-			helpTopic = *command
-		}
-		if helpTopic == "" {
-			// hacky way to make the CLI kick in as if the user typed `orchestrator -c help cli`
-			*command = "help"
-			flag.Args()[0] = "cli"
-		}
 	}
 
 	switch {
 	case helpTopic != "":
 		app.HelpCommand(helpTopic)
-	case len(flag.Args()) == 0 || flag.Arg(0) == "cli":
-		app.CliWrapper(*command, *strict, *instance, *destination, *owner, *reason, *duration, *pattern, *clusterAlias, *pool, *hostnameFlag)
```

Comment:
CLI is used in integration tests; a but like vitess' `endtoend` tests run command line `vtctl`. In that sense it's useful to keep. Unless you convert all integrated tests to run `orchestrator-client`, which is some undertaking.

Reply:
We will have to convert. I'll explain in the main answer.